modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/entries/scheduler/test/CheckObjectEntrySchedulerJobConfigurationTest.java

```diff
@@ -80,6 +80,43 @@ public static void setUpClass() throws Exception {
 				).build()));
 	}
 
+	@Test
+	public void testCheckObjectEntryDisplayDate() throws Exception {
+		Date date = new Date();
+
+		ObjectEntry objectEntry1 = ObjectEntryTestUtil.addObjectEntry(
```

Comment:
use an object with draft entry enabled. It's weird having an entry going from published to scheduled :thinking: